packages/web-components/src/components/promo-item/promo-item-statistic.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, property, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './promo-item.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The statistic content of promo item.
+ *
+ * @element dds-promo-item-statistic
+ */
+@customElement(`${ddsPrefix}-promo-item-statistic`)
+class DDSPromoItemStatistic extends LitElement {
+  /**
+   * The shadow slot this card heading should be in.
+   */
+  @property({ reflect: true })
+  slot = 'statistic';
+
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'heading');
+    }
+    if (!this.hasAttribute('aria-level')) {
+      this.setAttribute('aria-level', '4');
+    }
+    super.connectedCallback();
+  }
+
+  render() {
+    return html`
+      <slot></slot>
+    `;
+  }
+
+  // `styles` here is a `CSSResult` generated by custom WebPack loader
+  static styles = styles;
+}
+
+/* @__GENERATE_REACT_CUSTOM_ELEMENT_TYPE__ */
```

Comment:
Do you use React? Asking this question because I don't see this annotation in other components.

Reply:
@asudoh Nope, not at the moment, however, we might be in the future will need to generate a custom element type for React integration.